### PR TITLE
Install libssl1.1 in travis.sh

### PR DESCRIPTION
**Pull request by moskupols**
This should address #523

### CODE REVIEW

**Comment by henryiii** [comment]
Ping me when you have this working or if you need help. We can get the other PR in as soon as this gets cleaned up.

At some point, if I ever have "free" time, I need to migrate this over to GHA.

File: .travis.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ matrix:
     env: PYV=3.8
   - python: pypy3
     before_install:
-      apt-get install libssl1.1
+    - sudo apt-get install libssl1.1
```
Comment (henryiii) [inline .travis.yml]: I'd try sudo apt-get update && sudo apt-get install libssl1.1 - often the cache is empty in systems like this.

Comment (moskupols) [reply .travis.yml]: Sadly the problem is there's no libssl1.1 for xenial: the earliest ubuntu that has it in the standard repository is ubuntu 18 bionic.
Ubuntu 16 xenial is nearing its end of support.

**Comment by moskupols** [comment]
Even if I allow cryptography to link against the old version of OpenSSL, it still fails to build because of some redefinition between its sources and the headers of libssl.
Cryptography installation notes don't mention Ubuntu 16 among supported versions: https://cryptography.io/en/latest/installation.html .
So I'm afraid in order to test pypy builds with latest cryptography it may be required to drop testing for xenial.
I guess another solution would be to put a constraint cryptography<3.2.0?

**Comment by henryiii** [comment]
Then can't you bump the OS? Travis has bionic, I think? `dist: bionic`? Possibly for this one job for now?

File: .travis.yml [hunk 2]
```diff
@@ -16,6 +16,8 @@ matrix:
   - python: 3.8
     env: PYV=3.8
   - python: pypy3
```
Comment (henryiii) [inline .travis.yml:18]:
```suggestion
  - python: pypy3
    dist: bionic
```

IIRC this would work?

**Comment by moskupols** [comment]
@henryiii Yeah looks like that solved the problem

**Pull request merged**